From ac082cf1103661e249b186dcdbee8f5306fd9681 Mon Sep 17 00:00:00 2001
From: David Steele <dsteele@gmail.com>
Date: Wed, 18 Sep 2013 21:41:28 -0400
Subject: [PATCH 11/20] Raise keepalive period from 1 to 10 minutes.

Research suggests that 1-2 hours is fast enough.
(cherry picked from commit bfc706fdc5e39046ff03188bb2a67bd7839b98eb)
---
 cloudprint/cloudprint.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cloudprint/cloudprint.py b/cloudprint/cloudprint.py
index d4ee469..269710d 100755
--- a/cloudprint/cloudprint.py
+++ b/cloudprint/cloudprint.py
@@ -31,6 +31,12 @@ PRINT_CLOUD_URL = '/cloudprint/'
 # 'None' to poll only on startup and when we get XMPP notifications.
 POLL_PERIOD=30.0
 
+# wait period to retry when xmpp fails
+FAIL_RETRY=60
+
+# how often, in seconds, to send a keepalive character over xmpp
+KEEPALIVE=600.0
+
 LOGGER = logging.getLogger('cloudprint')
 LOGGER.setLevel(logging.INFO)
 #LOGGER.setLevel(logging.DEBUG)
-- 
1.8.4.rc3

